src/testUtils/mocks/entities.ts

```diff
@@ -1528,6 +1531,7 @@ function configureDividendDistribution(opts: DividendDistributionOptions): void
     expiryDate: opts.expiryDate,
     paymentDate: opts.paymentDate,
     details: dividendDistributionDetailsStub.resolves(opts.details),
+    getParticipant: dividendDistributionGetParticipantStub.resolves(opts.getParticipant),
```

Comment:
Since both these options objects are partial, you should first merge them with the default options

```
const details = { ...defaultDividendDistributionOptions.details, ...opts.details };
// ...
details: dividendDistributionDetailsStub.resolves(details),
```

Same for `getParticipant` (which should also have a default)